app/bundles/LeadBundle/Form/Type/FilterType.php

```diff
@@ -126,6 +126,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     $customOptions['multiple'] = true;
                     $type                      = 'choice';
                     break;
+                case 'lead_email_sent':
```

Comment:
As it perfectly fits the previous section - I suggest to just merge them, moving `case 'lead_email_sent':` directly after `case 'lead_email_received':`.